src/mem/ruby/system/GPUCoalescer.cc

```diff
@@ -599,6 +599,8 @@ GPUCoalescer::hitCallback(CoalescedRequest* crequest,
                 // data response is not needed.
                 case RubyRequestType_ATOMIC_NO_RETURN:
                     assert(pkt->isAtomicOp());
+                    data.popAtomicLogEntryFront();
+                    break;
```

Comment:
PS: added a break because I think there is no alternative fall through case when handling atomic packets -- correct me if I'm wrong.

Reply:
Ignore that, _ST case applies still, didn't notice the comment

Reply:
It does look like there needs to be a break here, thanks for catching.